app/views/analyse/help.scala

```diff
@@ -47,7 +47,9 @@ object help {
           isStudy option frag(
             header("Study actions"),
             row(frag(k("d")), trans.study.commentThisPosition()),
-            row(frag(k("g")), trans.study.annotateWithGlyphs())
+            row(frag(k("g")), trans.study.annotateWithGlyphs()),
+            row(frag(k("n")), "Next chapter"),
+            row(frag(k("p")), "Previous chapter")
```

Comment:
These should be made translateable. Just add the keys and the current text to `translation/source/study.xml` and add an entry to `modules/i18n/src/main/I18nKeys.scala` in the `study` object. Then you can do `trans.study.nextChapter()` similar to the lines above.